ntbs-service/Pages/Shared/_Breadcrumbs.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@using ntbs_service.Models
+@model List<Breadcrumb>
+
+
+@if (@Model != null)
+{
+    <nhs-breadcrumb>
+        <nhs-breadcrumb-list>
+            @foreach (var breadcrumb in Model)
+            {
+                <nhs-breadcrumb-item href=@breadcrumb.Url>@breadcrumb.Label</nhs-breadcrumb-item>
```

Comment:
this changes & to amp& in url and therefore causing error - not filling in data if id is not filled

Reply:
Not sure what this comment is trying to say..? Is there a sanitization problem here that we still need to resolve?